### DISCUSSION TITLE
How do i solve this anticheat broken/denied

### DISCUSSION THREAD

**Discussion by pirsonxyz** [Heroic-Games-Launcher/HeroicGamesLauncher#4219] [Q&A]
![image](https://github.com/user-attachments/assets/c89da738-a326-4495-b19a-b66e166a6e86)
I've been looking for the setting to disable this check and i couldn't find it

**Comment by GitUp1234** [comment]
![HGL_Advanced_Settings](https://github.com/user-attachments/assets/1411ddbb-1c28-4a59-a096-f387a2d7b080)

**Comment by zeroninefourfour** [comment]
This is stupid, they should just warning instead of blocking the game installs

**Reply by Vicepancake** [reply]
BUT I STILL CAN NOT PLAY MULTI PLAYER 

>:(
